custom_components/frigate/binary_sensor.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         self._obj_name = obj_name
         self._is_on = False
         self._frigate_config = frigate_config
+        self._attr_entity_registry_visible_default = obj_name != "all"
```

Comment:
So if this is false then it is `hidden` or `disabled` by default?

Reply:
It will be default enabled, and default visible (unless it's the `all`, and then it'll be enabled and hidden). This should be as described [here](https://github.com/blakeblackshear/frigate-hass-integration/issues/256#issuecomment-1133656578),  but let me know if you spot anything to be contrary.

Reply:
Right, I was mostly unsure what `_attr_entity_registry_visible_default = false` would do but that makes sense 👍